data/components/frame-list.js

```diff
@@ -182,9 +183,25 @@ var FrameBubble = React.createFactory(React.createClass({
     }
 
     if (!preview && frame.mqtt) {
+      var mqtt = frame.mqtt;
+      var data;
+      op = "MQTT " + mqtt.cmd;
+      if (mqtt.cmd === "publish") {
+        payload = mqtt.topic;
+        data = mqtt.payload;
+      } else {
+        payload = mqtt.messageId || mqtt.clientId || mqtt.returnCode;
+        data = {packet: mqtt};
+        if (mqtt.subscriptions)
+          data = {subscriptions: mqtt.subscriptions};
+        if (mqtt.granted)
+          data = {granted: mqtt.granted};
+        if (mqtt.unsubscriptions)
+          data = {unsubscriptions: mqtt.unsubscriptions};
+      }
       preview = TreeView({
         key: "preview-mqtt",
-        data: {"MQTT": frame.mqtt},
+        data: data,
```

Comment:
My thinking was that it would not be so important to be consistent between protocols, because all the messages in a connection adhere to the same protocol. Here it doesn't make sense to display the unparsed websocket frame payload so instead, I tried to display as much parse results as possible. Instead of having to click "MQTT"->"payload", you could see the parsed payload content of each message directly. Along the same lines, frame type is shown as "MQTT "+command instead of all frames displaying "BINARY".

Having said all that, it is inconsistent unless the other protocols are changed similarly, so I could split this `data` logic (lines 191, 194-200, 204) into another branch.